i18n/de.json

```diff
@@ -337,5 +337,19 @@
 	"srf-spreadsheet-link": "Tabellenkalkulation",
 	"srf-paramdesc-spreadsheet-filename": "Das zu erzeugende Dateiformat. Erlaubte Werte: xlsx, xls, ods, csv. Standard: xlsx",
 	"srf-paramdesc-spreadsheet-templatefile": "Der Name einer Tabellenkalkulationsdatei aus dem Namensraum ''Datei'', die zur Formatierung der erzeugten Datei verwendet wird.",
-	"srf-paramdesc-icalendar-timezone": "Legt eine durch Kommata getrennte Liste mit Zeitzonen fest"
+	"srf-paramdesc-icalendar-timezone": "Legt eine durch Kommata getrennte Liste mit Zeitzonen fest",
```

Comment:
Usually there is no need to provide I18n other than en and qqq, however since it is there it's ok.